tests/cosmosLightClient/data/store_data_after_all_updates.txt

```diff
@@ -0,0 +1 @@
1,79,145,191,148,21,8,14,88,77,151,140,34,34,229,247,28,179,48,86,2,164,178,67,23,95,217,106,228,137,149,230,54,137,165,105,84,164,148,53,18,192,40,178,225,153,86,16,104,150,129,177,121,92,187,119,127,239,223,62,251,21,159,233,55,81,36,124,232,177,27,253,117,106,76,212,12,112,67,227,175,4,32,94,147,28,98,90,172,63,67,180,209,131,116,62,86,236,139,173,204,208,248,158,211,138,150,83,220,78,49,82,156,247,77,53,132,198,235,248,171,52,97,214,127,46,24,48,98,74,120,139,105,158,141,204,134,162,134,153,231,90,202,43,202,158,144,177,245,70,42,195,252,178,56,27,155,144,104,123,111,201,191,161,21,157,235,138,8,85,251,122,225,168,185,20,97,8,199,17,44,105,112,161,53,30,93,209,224,63,207,74,129,235,168,254,72,209,88,240,205,116,155,8,109,190,215,18,26,113,197,117,52,144,207,199,232,197,233,46,116,174,24,140,63,82,230,246,251,164,139,177,114,169,231,243,98,6,26,150,26,142,177,34,6,67,200,85,213,221,153,134,233,52,145,159,28,31,110,120,39,2,253,209,237,213,3,116,122,44,132,10,167,155,173,173,20,5,121,146,90,70,147,204,1,187,47,159,242,74,135,238,136,236,61,230,51,107,210,137,85,88,8,47,151,15,194,178,191,246,62,12,194,94,195,17,105,67,147,143,122,106,69,192,139,183,171,137,38,127,13,154,136,114,123,249,197,210,170,225,148,62,104,228,22,241,8,53,55,138,197,95,21,204,16,75,78,231,150,217,177,140,251,240,246,106,9,13,107,4,87,176,32,82,190,85,169,76,7,32,192,168,111,117,151,222,169,87,255,219,161,197,232,25,4,241,16,1,251,162,200,86,122,69,92,76,166,132,99,212,169,195,204,117,59,44,41,170,197,238,83,4,50,24,193,21,170,183,68,181,182,35,241,57,178,113,44,83,163,210,227,49,113,8,249,107,153,190,11,82,79,100,141,58,16,28,236,65,109,166,0,110,131,248,194,166,83,153,110,5,87,127,44,75,197,38,249,162,200,202,75,84,149,124,172,138,99,243,78,131,67,9,159,147,52,74,192,47,30,143,53,14,52,14,22,186,139,187,98,72,23,122,89,244,112,46,197,76,122,85,135,150,171,104,242,133,28,118,198,204,164,230,146,226,195,108,216,62,8,190,111,35,32,103,40,14,135,232,126,58,2,223,145,192,17,138,230,243,155,223,241,67,181,21,159,213,255,237,53,128,33,182,179,8,235,120,200,202,147,3,104,22,150,6,57,231,26,205,130,92,132,240,11,118,140,111,26,57,134,151,15,237,178,156,184,162,180,25,89,53,182,16,70,238,68,50,167,22,150,99,7,36,179,150,121,200,163,214,240,105,68,12,56,184,138,250,34,163,136,61,243,160,138,8,224,202,12,234,100,153,116,51,28,205,166,33,158,153,82,40,212,58,126,48,233,140,85,131,157,152,55,49,10,50,153,153,218,154,251,40,142,40,184,248,226,97,11,228,101,30,89,241,80,26,240,161,196,15,113,5,105,202,215,48,150,1,143,215,73,160,158,193,199,135,102,128,153,193,191,15,113,167,234,160,66,0,0,0,0,0,208,217,2,0,0,0,0,0,27,7,59,240,170,94,251,246,183,83,148,10,11,79,233,155,206,26,138,106,181,137,134,17,121,100,164,103,248,209,226,184,197,158,138,249,80,238,227,206,34,3,12,109,10,153,87,242,135,109,213,200,204,26,61,64,7,70,226,147,236,63,167,109,226,188,104,23,138,219,248,58,42,3,205,6,175,243,136,221,126,45,106,108,165,138,159,210,146,249,225,13,34,60,54,239,0,2,0,0,0,0,0,0,0,0,0,0,0,0,0,0
```

Comment:
Instead of comparing the whole store data against a precise value, can't we query the state of the contract and obtain pieces of information that are certain to remain the same such as the latest block header?

Using this file will create a more fragile setup where every time we change the contract implementation details (what is being stored on chain), we'll have to regenerate this opaque file.

Reply:
My motivation was to be sure everything is correct, but I agree it is overkill. Let's stick with the check of the latest block header